src/Translator/assets/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * PHP strtr's equivalent, inspired and adapted from https://stackoverflow.com/a/37949642.
+ * @param string The string to replace in
+ * @param replacePairs The pairs of characters to replace
+ */
+export function strtr(string: string, replacePairs: Record<string, string | number>): string {
```

Comment:
This function and the one in `src/Translator/assets/dist/translator_controller.js` seems identical, can they be merged?

Reply:
Dist folder, never mind.